src/components/discussion-link.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * Discussion Link
+ */
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const DiscussionLink = ({linkType, link}) => {
+  if (link && link != "") {
+    return (
+      <a
+        target="_blank"
+        rel="noreferrer noopener"
+        href={link}
+        style={{"text-decoration": "underline"}}
+      >
+        <p>Discuss further on {linkType}</p>
```

Comment:
Do we have tracking by default on if users click these?

Reply:
This should do it: a05d8a9

Reply:
This doesn't work. There is another way of doing it but I am having some problems with it. Is it ok if we remove this commit and deploy it without the analytics and I add this later?

Reply:
Yo, just to check, you didn't manage to fix this in the end? So it no analytics on those links?

Reply:
Yeah, I didn't fix it and ai removed the commit, so there are no events being sent to google analytics for clicks on this link. I need to refactor the analytics system because we are currently using out-of-date packages.